src/pages/comparison/comparison.tsx

```diff
@@ -239,24 +277,29 @@ export const Comparison = ({ mode }: ComparisonPageProps) => {
   };
 
   const downloadClickHandler = () => {
-    const data = {
-      viewState: viewState.main,
-      datasets: [
-        {
-          ...loadManagerRef.current.leftStats,
-          ellapsedTime: loadManagerRef.current.leftLoadingTime,
-        },
-        {
-          ...loadManagerRef.current.rightStats,
-          ellapsedTime: loadManagerRef.current.rightLoadingTime,
-        },
-      ],
-    };
-    downloadJsonFile(data, "comparison-stats.json");
+    if (!bookmarks.length) {
+      const data: StatsData = {
+        viewState: viewState.main,
+        datasets: [
+          {
+            ...loadManagerRef.current.leftStats,
+            ellapsedTime: loadManagerRef.current.leftLoadingTime,
+          },
+          {
+            ...loadManagerRef.current.rightStats,
+            ellapsedTime: loadManagerRef.current.rightLoadingTime,
+          },
+        ],
+      };
+      downloadJsonFile(data, "comparison-stats.json");
+      return;
+    }
+    downloadJsonFile(bookmarksStats, "bookmarks-stats.json");
```

Comment:
```suggestion
    downloadJsonFile(comparisonStats, "comparison-stats.json");
```

Reply:
We can treat one-view-state comparison stats as array of 1 element to unify the format.